packages/map-template/src/components/Search/Search.jsx

```diff
@@ -316,75 +317,49 @@ function Search({ onSetSize, isOpen }) {
         }
     }, [useKeyboard]);
 
-
     return (
         <div className="search"
             ref={searchRef}
-            style={calculateContainerStyle()}
-            >
-
-                { /* Search field that allows users to search for locations (MapsIndoors Locations and external) */ }
```

Comment:
Aww, I liked the whitespace that made it easier to read 😢 

Did you change anything else but removing empty lines?

Reply:
I knoow... I liked the readability as well, but it's inconsistent with everything else and also a bit hard to maintain when the formatter goes against me 🙃

Reply:
Nothing else was changed, except the empty lines.

Reply:
Don't let the editor win against you 😄
It's not inconsistent with everything else: look at LocationDetails.jsx where we have a similar-looking pattern for readability. Also, we need to make the code easier to read.

Reply:
If you just leave a little white space for readability, I'm happy 😄